chart/templates/crds/druid.apache.org_druids.yaml

```diff
@@ -1329,6 +1340,16 @@ spec:
                               type: string
                           type: object
                       type: object
+                    deploymentConfig:
+                      properties:
+                        batchSize:
+                          format: int32
+                          type: integer
+                        mode:
+                          type: string
+                      required:
+                      - mode
```

Comment:
If we can set mode to a default value rather than making it required, it would make it easier to transition.